tests/test_jx/test_deep_ops.py

```diff
@@ -636,7 +636,7 @@ def test_setop_w_complicated_where(self):
 
     def test_id_select(self):
         """
-        ALWAYS GOOD TO HAVE AN ID, CALL IT "_id"
```

Comment:
We should not need to change the test.  `__guid__` in the database should be named `_id` from the snowflake perspective.